database/db.sql

```diff
@@ -71,7 +77,13 @@ ALTER TABLE `users`
 --
 -- AUTO_INCREMENT for table `users`
 --
-ALTER TABLE `users`
+altar TABLE `users`
+  MODIFY `id` int UNSIGNED NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3;
```

Comment:
Since this is a new table, it is better you set id primary key and auto_increment in TABLE users definition rather than altering the Table